Add dedupCreate/dedupExercise helpers to the DAML trigger API (#3280)
digital-asset/daml: docs/source/triggers/index.rst

Finally, we also need to create copies that do not already exists. We
want to avoid creating copies for which there is already a command in
flight. The DAML Trigger API provides a ``dedupCreate`` helper for this

> which only sends the commands if it is not already in flight.

Reviewer: ```suggestion
which only sends the commands that are not already in flight.
```
